sdk/schema/credname/names.go

```diff
@@ -13,8 +13,8 @@ const (
 	AuthToken           = sdk.CredentialName("Auth Token")
 	CLIToken            = sdk.CredentialName("CLI Token")
 	Credential          = sdk.CredentialName("Credential")
-	Credentials         = sdk.CredentialName("Credentials")
```

Comment:
Was this previously unused? Because it looks like the `Credentials` type was there before this PR

Reply:
Only references/uses of the constant seem to be in the file where it's defined, `sdk/schema/credname/names.go`. To double-check, a global search for `credname.Credentials` also gives no results.

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/53918631/218535386-b023f23c-a2e3-444b-b624-c3856f8760bc.png">

Reply:
Perfect!